contribs/gmf/src/directives/datepicker.js

```diff
@@ -0,0 +1,170 @@
+goog.provide('gmf.DatePickerDirective');
+goog.provide('gmf.DatePickerController');
+
+goog.require('gmf');
+
+gmf.module.value('gmfDatePickerTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     * @return {string} Template URL.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfDatePickerTemplateUrl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/datepicker.html';
+    });
+
+
+/**
+ * Provide a directive to select a signle date or a range of dates
+ * @param {string|function(!angular.JQLite=, !angular.Attributes=)}
+ * gmfDatePickerTemplateUrl Template for the directive.
+ * @param  {angular.$timeout} $timeout angular timeout service
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfDatePicker
+ */
+gmf.DatePicker = function(gmfDatePickerTemplateUrl,  $timeout) {
+  return {
+    scope : {
+      onDateSelected : '&',
+      time : '='
+    },
+    bindToController : true,
+    controller : 'gmfDatePickerController',
+    controllerAs : 'timeCtrl',
+    restrict: 'AE',
+    templateUrl : gmfDatePickerTemplateUrl,
+    link: function(scope, element, attrs, ctrl) {
+
+      /** @type {string} */
+      var dFormat;
+
+      var lang =  ctrl.gettextCatalog_.getCurrentLanguage();
+      $['datepicker']['setDefaults']($['datepicker']['regional'][lang]);
+
+      switch (lang) {
+        case 'de':
+          dFormat = 'dd.mm.yy';
+          break;
+        default:
+          dFormat = 'dd/mm/yy';
+          break;
+      }
```

Comment:
@sbrunner date format is used by the component jquery-ui datepicker itself. 

- http://api.jqueryui.com/datepicker/#option-dateFormat
- http://api.jqueryui.com/datepicker/#utility-formatDate

Therefore inputs are filled by the the jquery datepicker itself. I don't need to use the Angular filter.

As discussed with @ybolognini, for now:

- dd.mm.yy format for 'de'
- otherwise  dd/mm/yy